main.go

```diff
@@ -340,7 +339,7 @@ func main() {
 		VersionCodes: versionCodes,
 	}
 
-	if configs.Track == rolloutTrackName {
+	if configs.UserFraction != "" {
 		userFraction, err := strconv.ParseFloat(configs.UserFraction, 64)
```

Comment:
this will not be required AFAIK

Reply:
If it is def value, it should not be assigned.